internal/gribi/gribi.go

```diff
@@ -218,6 +218,14 @@ func (c *Client) AddNH(t testing.TB, nhIndex uint64, address, instance string, e
 			NH = NH.WithNextHopNetworkInstance(opt.VrfName)
 		}
 		c.fluentC.Modify().AddEntry(t, NH)
+	case "NHvrf":
```

Comment:
In #1181 name "VRFOnly" is introduced, can we use the same here (just to avoid future merge conflicts or having two separate names for the same case)?

Reply:
I have addressed this.